bindings/rust/src/bindings/mod.rs

```diff
@@ -19,6 +19,20 @@ pub const BYTES_PER_G2_POINT: usize = 96;
 /// 65 is fixed and is used for providing multiproofs up to 64 field elements.
 const NUM_G2_POINTS: usize = 65;
 
+/// A trusted (valid) KZG commitment.
```

Comment:
Hmm, can you explain a bit more how this works? We have `KZGCommitment` (in `bindings.rs`) and `KzgCommitment` here? When to use one and when the other? How to avoid confusion?

Reply:
`KzgCommitment`  will be used in the public API, and `KZGCommitment` in the FFI boundary. Fortunately the compiler will complain loudly is used one instead of the other and only us need to deal with the existence of both types, not the final user, since the `KZG` one is not exposed